ArtHub/Controllers/ProfileArtController.cs

```diff
@@ -68,7 +71,7 @@ public async Task<IActionResult> UpdateArt(int profileId, int artId, CreateArtDa
         // To protect from overposting attacks, see https://go.microsoft.com/fwlink/?linkid=2123754
         [Authorize]
         [HttpPost]
-        public async Task<ActionResult<Art>> CreateArt(int profileId, CreateArtData art)
+        public async Task<ActionResult<Art>> CreateArt(int profileId, CreateArtData art, IFormFile artImage)
```

Comment:
Might wanna change artImage to artFile so it's consistent with the front-end name.